### ISSUE TITLE
Missing metadata property on some objects

### ISSUE THREAD

**Issue by kevinvalk** [mollie/mollie-api-python#172]
On the objects
- customer
- order
- payment
- refund

There is a property to access the metadata
```python
@property
def metadata(self):
    return self._get_property('metadata')
```

However, all other objects that do have metadata according the API documentation (example: https://docs.mollie.com/reference/v2/subscriptions-api/get-subscription) do **NOT** have this property.

I believe this is an inconsistency as I am able to access the underlying metadata using:
```python
metadata = subscription._get_property("metadata")
```

Hopefully you can add the metadata property to all objects that have metadata attached for consistency and clarity.

Thanks!

**Comment by whyscream** [comment]
Hi @kevinvalk , thanks for reporting. We'll add the missing properties

**Comment by whyscream** [comment]
@kevinvalk We just released v2.4.1, with support for all missing metadata properties that we could find. See: https://pypi.org/project/mollie-api-python/2.4.1/
